### ISSUE TITLE
Solve the problem with fonts by simply using XeLaTex or LuaLaTeX

### ISSUE THREAD

**Issue by dietmarw** [USN-latex/Beamer-Template#1]
Nice template. I noticed you didn't want to use the Calibri font because of pdflatex incompatibility. 
Now pdflatex is outdated anyway and one should use XeLaTeX or LuaLaTeX instead (LuaLaTeX actually works better for beamer if you have videos included). These two are pretty much standard in all LaTeX distributions. If you do so then you can easily define the usage of Carlito (the free Calibri equivalent). I've done so in my thesis template: https://github.com/dietmarw/USN-LaTeX/blob/master/Theses/USN-base.cls#L52

**Comment by einhalv** [comment]
I just noticed your comment now when updating to USN. I will look into it.

**Comment by dietmarw** [comment]
Probably a reason to keep this issue open until it is fixed, right?

**Comment by einhalv** [comment]
Yes, probably. I am a total rookie on github and didn't quite realize what I was doing. The new style manual for the university defines Times New Roman and Calibri as the fonts for use with Office products apparently based on the belief that these tools are the only ones used to produce material internally. It is tempting to use this narrowmindedness  as an excuse to keep things as is, but on the other hand it would be good if the LaTeX template is close to the Office template.  Hence, I reopen and will look into it (when I find the time).

**Comment by dietmarw** [comment]
Simply use Carlito which is the exact same shape and metric compatible. See https://github.com/dietmarw/USN-LaTeX/blob/master/Theses/USN-base.cls#L51-L52
